src/plugins/intel_npu/tools/protopipe/src/simulation/layers_reader.cpp

```diff
@@ -31,6 +31,17 @@ static std::string getModelFileName(const InferenceParams& params) {
     ASSERT(false);
 }
 
+static void adjustLayerPartialShapes(LayersInfo& layers) {
+    for (auto& layer : layers) {
+        for (auto& dim : layer.dims) {
+            if (dim == -1) {
+                LOG_INFO() << "Dynamic partial shape detected for layer " << layer.name << " partial shape will be set to 1" << std::endl;
```

Comment:
It's not shape but dimension

Reply:
Btw, we don't change model shape. 

The message can be something like: 
```
Dynamic shape: [-1, 32, 256, 256] has been detected for layer <NAME>. Data with shape [1, 32, 256, 256] will be provided as input for this layer.
```

Reply:
Done